omnibus/config/software/chef.rb

```diff
@@ -44,6 +44,9 @@
 dependency "chef-gem-byebug"
 dependency "chef-gem-debug_inspector"
 dependency "chef-gem-binding_of_caller"
+dependency "chef-gem-rbnacl-libsodium" unless ios_xr?
+dependency "chef-gem-bcrypt_pbkdf-ruby" unless solaris?
```

Comment:
and likely you just want to make this an `unless ios_xr? || solaris?` around both of them (or all three)